spec/client_mcm_spec.rb

```diff
@@ -37,6 +39,10 @@ def safe_user_id(name)
     @client.assign_user_id(@user_id, @cluster_name)
   end
 
+  it 'should assign user ids to a cluster' do
+    @client.assign_user_ids([@user_id_0, @user_id_1], @cluster_name)
```

Comment:
Maybe here you need an expectation? Like on the CTS, to make sure users were indeed assigned.

Reply:
done